packages/signer/signer-btc/src/internal/app-binder/command/utils/BtcDerivationPathUtils.ts

```diff
@@ -0,0 +1,59 @@
+const MAX_INDEX = 0x7fffffff;
+export class BtcDerivationPathUtils {
+  /**
+   * Parses a BIP32 path string into an array of numbers.
+   * @param path - The BIP32 path string (e.g., "m/44'/0'/0'/0/0" or "44'/0'/0'/0/0").
+   * @returns An array of numbers representing the derivation steps, or an empty array if the path is invalid.
+   */
+  static parseBip32(path: string): number[] {
+    if (!path) return [];
+    let trimmedPath = path.trim();
+    if (trimmedPath.startsWith("m/")) {
+      trimmedPath = trimmedPath.slice(2);
+    } else if (trimmedPath === "m") {
+      return [];
+    }
+    if (trimmedPath === "") return [];
+    const components = trimmedPath
+      .split("/")
+      .filter((component) => component.length > 0);
+    const parsedPath = components.map((component) => {
+      let hardened = false;
+      let indexStr = component;
+      if (component.endsWith("'")) {
+        hardened = true;
+        indexStr = component.slice(0, -1); // Remove the "'" character
+      }
+      const index = parseInt(indexStr, 10);
+      if (isNaN(index) || index < 0 || index > MAX_INDEX) return null;
+      if (hardened) {
+        return index + 0x80000000;
+      } else {
+        return index;
+      }
+    });
+    if (parsedPath.includes(null)) return [];
+    return parsedPath as number[];
+  }
+  /**
+   * Encodes an array of BIP32 path steps into a big-endian Uint8Array.
+   * @param bip32Steps - An array of numbers representing the BIP32 path steps.
+   * @returns A Uint8Array where each step is represented as 4 big-endian bytes.
+   */
+  static encodeBip32Path(bip32Steps: number[]): Uint8Array {
```

Comment:
I'm not sure that it's useful, we add derivation path to data with apdu builder helpers in other signers

```suggestion
    const path = BtcDerivationPathUtils.splitPath(derivationPath);
    path.forEach((element) => {
      builder.add32BitUIntToData(element);
    });
```